src/viz-probation-population/VizProbationPopulation.js

```diff
@@ -0,0 +1,172 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import judicialDistrictsTopology from "../assets/maps/us_nd_judicial_districts.json";
+import {
+  DEFAULT_TENANT,
+  OTHER_LABEL,
+  TENANTS,
+  TOTAL_KEY,
+  OTHER,
+} from "../constants";
+import PopulationViz from "../population-viz";
+import StateMap from "../state-map";
+import { recordIsTotal } from "../utils";
+
+const ASPECT_RATIO = TENANTS[DEFAULT_TENANT].aspectRatio;
+
+const ProbationMapWrapper = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+const ParticipantCount = styled.text`
+  dominant-baseline: middle;
+  fill: ${(props) =>
+    // eslint-disable-next-line no-nested-ternary
+    props.active
+      ? props.theme.maps.default.fill
+      : props.hover
+      ? props.theme.colors.highlight
+      : props.theme.colors.programParticipation};
+  font: ${(props) => props.theme.fonts.displayMedium};
+  font-size: 20px;
+  pointer-events: none;
+  text-anchor: middle;
+`;
+
+const OtherWrapper = styled.button`
+  background: none;
+  border: 0;
+  color: ${(props) =>
+    props.active ? props.theme.colors.highlight : props.theme.colors.heading};
+  cursor: pointer;
+  padding: 16px;
+  margin-left: -16px;
+  text-align: center;
+
+  &:hover {
+    color: ${(props) => props.theme.colors.highlight};
+  }
+`;
+
+const OtherLabel = styled.div`
+  font: ${(props) => props.theme.fonts.body};
+`;
+
+const findDistrictRecord = (district) => (record) =>
+  record.district === district;
+
+function ProbationMap({ currentLocation, data, onLocationClick, width }) {
+  // eslint-disable-next-line react/prop-types
+  const DistrictLabel = ({ hover, locationId, objectId }) => {
```

Comment:
Nit: `districtId` may be a better name than `objectId`

Reply:
`objectId` is the more generic prop that `StateMap` will pass in here ... I was waffling on whether to call it something slightly less generic like `regionId` but it is quite literally just the id of a topology `object` from the topojson file. it's not even necessarily guaranteed to be a region on a map, it could just be a point or whatever. `topologyObjectId`? is that better?